communication-protocols/LoRaWAN/config.json

```diff
@@ -1,8 +1,92 @@
 {
+    "identifier": "devEUI",
     "fields": [
-        {"label":"Dev EUI", "value":"devEUI", "isRequired": true},
-        {"label":"App EUI", "value":"appEUI", "isRequired": true},
-        {"label":"App Key", "value":"appKey", "isRequired": true}
-    ],
-    "identifier": "devEUI"
-}
+        {
+            "label": "Dev EUI",
+            "value": "devEUI",
+            "isRequired": true,

```

Comment:
This can be removed, it is the same as the fisrt validation rule right ?

Reply:
I prefer to leave it for now so we can function with both approaches at the same time for a while